docs/config-file-resolutions-yml.md

```diff
@@ -0,0 +1,94 @@
+
+To be able to show why a resolution is acceptable, each resolution must include an explanation. 
+The explanation consists of:
+
+* `reason` -- quick indicator why the issue was resolved. Must be selected from a predefined list of options. 
+* `comment` -- free text that provides an additional explanation and maybe a link for further information
+
+## Resolving Errors
+
+If the ORT results show errors, the best approach is usually to fix them and run the scan again. 
+However, sometimes it is not possible, for example if error occurs in the license scan
+of a third-party dependency which cannot be fixed or updated.
+
+In such situations, you can *resolve* the error in any future scan by adding a resolution
+to the `resolutions.yml` to mark it as acceptable.
+
+The code below shows the structure of an error resolution in the `resolutions.yml` file:
+
+```yaml
+errors:
+- message: "A regular expression matching the error message."
+  reason: "One of ErrorResolutionReason e.g. BUILD_TOOL_ISSUE,CANT_FIX_ISSUE,SCANNER_ISSUE."
+  comment: "A comment further explaining why the reason above is acceptable."
+```
+where the list of available options for `reason` is defined in
+[ErrorResolutionReason.kt](../model/src/main/kotlin/config/ErrorResolutionReason.kt)
+
+For example, to ignore an error related to a build tool problem, your `resolutions.yml` could include:
+
+```yaml
+errors:
+- message: "Does not have X.*"
+  reason: "BUILD_TOOL_ISSUE"
+  comment: "Error caused by a known issue for which fix is being implemented, see https://github.com/..."
+```
+
+## Resolving Policy Rule Violations
+
+Resolutions should not be not used to resolve license policy violations as they do not change generated open notices.
+To resolve a license policy violation either add a local license finding curation
+to the [.ort.yml file](./config-file-ort-yml.md) if the finding is in your code repository or add a curation to the [curations.yml](config-file-curations-yml.md) if the violation occurs in a third-party dependency.
+
+The code below shows the structure of an policy violation resolution in the `resolutions.yml`file:
+
+```yaml
+rule_violations:
+- message: "A regular expression matching the rule violation message."
+  reason: "One of RuleViolationResolutionReason e.g. CANT_FIX_EXCEPTION, DYNAMIC_LINKAGE_EXCEPTION."
+  comment: "A comment further explaining why the reason above is applicable."
+```
+
+where the list of available options for `reason` is defined in [RuleViolationResolutionReason.kt](../model/src/main/kotlin/config/RuleViolationResolutionReason.kt)
+
+For example, to confirm your organization has acquired an org-wide Qt commercial license, your `resolutions.yml` could include:
+
+```yaml
+errors:
+- message: ".*LicenseRef-scancode-qt-commercial-1.1 found in 'third-party/qt/LICENSE'.*"
+  reason: "LICENSE_ACQUIRED_EXCEPTION"
+  comment: "Org-wide commercial Qt license was purchased, for details see https://jira.example.com/issues/SOURCING-1234"
+```
```

Comment:
This whole file seems to be a copy of a part of the docs for `.ort.yml`. Instead of repeating it, you should reference this document from the docs for `.ort.yml`.

Reply:
Won't fix: If you pay close attention you notice that the .ort.yml example is project-specfic whilst this one is org-wide example to show the different between .ort.yml and resolutions.yml resolutions